adoc/chapters/programming_interface.adoc

```diff
@@ -23273,10 +23267,9 @@ corresponding [code]#vec#.
 
 === Native precision math functions
 
-In SYCL the implementation-defined precision math functions are defined in the
-namespace [code]#sycl::native#.
-The functions that are available within this namespace are specified in
-<<table.native.math.functions>>.
+<<table.native.math.functions>> describes the native precision math functions
```

Comment:
Here we describe the functions in the `native` namespace as native precision math functions, but if we are removing the precision requirement of the regular math functions, this term becomes less meaningful. Maybe we should introduce some wording which states that the regular math functions are intended to be precise, whereas the native math functions are intended to be faster at the cost of precision, even as a non-normative note?

Reply:
Yes, I think that makes sense.  I can add a paragraph similar to the one I added below for the half-precision functions.  I'll have to think how to differentiate the half-precision functions from the native-precision ones, though.  They are both defined by the backend and both trade off precision for performance.

Reply:
I addressed this in 56196d2c528cf71c86555f40b4dfd96a502757b4.